rpc/services/peer-rpc-svc.go

```diff
@@ -39,3 +48,56 @@ func (p *PeerService) ValidateAdd(args *RPCPeerAddReq, reply *RPCPeerAddResp) er
 
 	return nil
 }
+
+func storeEtcdEnv(env *RPCEtcdEnvReq) error {
```

Comment:
Is this a function which is for services? I think this can definitely move to utils.